tasks/system_probe.py

```diff
@@ -833,10 +833,10 @@ def build_object_files(
     ctx.run(f"ninja -d explain -f {nf_path}")
 
     if not windows:
-        runner = ctx.run if is_root() else ctx.sudo
-        runner(f"mkdir -p {EMBEDDED_SHARE_DIR}")
-        runner(f"cp -R {build_dir} {EMBEDDED_SHARE_DIR}")
-        runner(f"chown root:root -R {EMBEDDED_SHARE_DIR}")
+        sudo = "" if is_root() else "sudo"
+        ctx.run(f"{sudo} mkdir -p {EMBEDDED_SHARE_DIR}")
+        ctx.run(f"{sudo} cp -R {build_dir} {EMBEDDED_SHARE_DIR}")
```

Comment:
Don't you need the `*` ?

Reply:
Yes, that was the original fix that was overwritten by me cherry-picking the sudo fix from another branch I have -- thanks for catching that.